pandas/tests/indexes/multi/test_constructors.py

```diff
@@ -5,8 +5,11 @@
 import itertools
 
 import numpy as np
+import pyarrow as pa
```

Comment:
This needs to be imported in the test itself (its an optional dependency not installed on all builds)